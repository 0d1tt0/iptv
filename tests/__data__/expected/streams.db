{"title":"ATV","id":"ATV.ad","filepath":"tests/__data__/input/channels/ad.m3u","resolution":{"height":720,"width":null},"status":{"label":"Offline","code":"offline","level":5},"url":"https://iptv-all.lanesh4d0w.repl.co/andorra/atv","http":{"referrer":"","user-agent":""},"is_broken":true,"updated":false,"cluster_id":1,"_id":"verufR2ehwdsfou3"}
{"title":"Fox Sports 2 Asia (Thai)","id":"FoxSports2AsiaThai.us","filepath":"tests/__data__/input/channels/us_blocked.m3u","resolution":{"height":720,"width":null},"status":{"label":"","code":"online","level":1},"url":"https://example.com/playlist.m3u8","http":{"referrer":"","user-agent":""},"is_broken":false,"updated":true,"cluster_id":1,"_id":"sLG04kZhqlEcYc25"}
