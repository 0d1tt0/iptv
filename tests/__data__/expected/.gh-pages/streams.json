[{"channel":"LDPRTV.ru","title":"ЛДПР ТВ","url":"http://46.46.143.222:1935/live/mp4:ldpr.stream/playlist.m3u8","is_online":false,"http_referrer":"","user_agent":""},{"channel":"BBCNews.uk","title":"BBC News HD","url":"http://1111296894.rsc.cdn77.org/LS-ATL-54548-6/index.m3u8","is_online":true,"http_referrer":"","user_agent":""},{"channel":"AndorraTV.ad","title":"ATV","url":"https://iptv-all.lanesh4d0w.repl.co/andorra/atv","is_online":false,"http_referrer":"","user_agent":""},{"channel":"BBCNewsHD.ad","title":"BBC News HD","url":"https://master.starmena-cloud.com/hls/libyas.m3u8","width":1024,"height":576,"bitrate":0,"is_online":true,"http_referrer":"","user_agent":""},{"channel":"KayhanTV.af","title":"Kayhan TV","url":"http://208.93.117.113/live/Stream1/playlist.m3u8","is_online":false,"http_referrer":"","user_agent":""},{"channel":"Sharq.af","title":"Sharq","url":"https://forerunnerrtmp.livestreamingcdn.com/output18/output18.stream/playlist.m3u8","width":1280,"height":720,"bitrate":2226543,"is_online":true,"http_referrer":"","user_agent":""}]